ui/app/webworker.js

```diff
@@ -62,6 +63,7 @@ const agent = async ({ ask, message, type, tell }) => {
       }
       case 'sys/attach':
         self.id = id;
+        sys.setConfig(config);
```

Comment:
Do I need to make these changes to my worker to keep it up to date?

Reply:
Only if you want to send config to the workers -- it'll use the compiled in defaults otherwise.

Reply:
Very nice.